src/style.css

```diff
@@ -0,0 +1,76 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  margin: 0;
+}
+
+.header {
+  max-width: 100%;
```

Comment:
```suggestion
  width: 100%;
```

Reply:
I wanted to make a content-limiting container. So, now it's only got in the way